content/posts/2023-11-07-dta-oriented-blogging.dj

```diff
@@ -13,13 +13,13 @@ be enough, often it isn't.
 
 If a blog has multiple pages, you usually want to have some common elements --- header, footer,
 style, etc. It _is_ possible to get common style by copy-pasting an existing page every time you
-need to add something. This make changes hard --- having consistent layout at a single point in time
+need to add something. This makes changes hard --- having consistent layout at a single point in time
 is not sufficient, one almost always wants to be able to apply consistent modifications as well.
 
 The second issue with hand-written html is that some parts might be very annoying to hand-write. For
 example, code snippets with syntax highlighting require quite a few tags.
 
-Finally, writing html by hand is not necessary most convenient. A `*` bullet-list certainly is more
```

Comment:
Ah, thanks for this one in particular, it’s only recently that I’ve realized that “necessary” and “necessarily” exist as two different words!